docs/buildah-build.1.md

```diff
@@ -327,9 +336,10 @@ Run up to N concurrent stages in parallel.  If the number of jobs is greater tha
 stdin will be read from /dev/null.  If 0 is specified, then there is
 no limit on the number of jobs that run in parallel.
 
-**--label** *label*
+**--label** *label[=value]*
 
 Add an image *label* (e.g. label=*value*) to the image metadata. Can be used multiple times.
+If *label* is named, but neither `=` nor a `value` is provided, then the *label* is removed from the image.
```

Comment:
@nalind not sure looks like this breaks exisiting podman tests. `--label xyz` will not get applied and some podman tests are directly using that.  I see some failures here: https://github.com/containers/podman/pull/14118

Reply:
Opened #3963.